components/fxa-client/src/fxa_client.udl

```diff
@@ -528,7 +528,12 @@ interface FirefoxAccount {
   //
   [Throws=FxaError]
   void send_single_tab([ByRef] string target_device_id, [ByRef] string title, [ByRef] string url );
-  
+
+
+  // ...
```

Comment:
nano-nit from the future - triple-slashes in UDL will cause the comments to be emitted as docstrings. Nothing consumes them yet, and no one else does this yet, so not at all clear this should, but at some point we might as well start doing that :)

Reply:
Done! Everything should have `///` docs now.